### PR TITLE
Mongodb

### PR DESCRIPTION
**Pull request by Satttoshi**
pls checkk
We added 

image

### CODE REVIEW

**Review by ZeshanShahid79 — APPROVED** [review]
Great work, espacially the Put Request

**Review by GoekhanAgkurt — APPROVED** [review]
looks amazing 🚀 send it to the moon.

File: backend/src/test/java/de/neuefische/backend/IdServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.data.annotation.Id;
+
+import static org.mockito.Mockito.when;
+
+class IdServiceTest {
+
+    private final IdService idService = Mockito.mock(IdService.class) ;
```
Comment (toshydev) [inline backend/src/test/java/de/neuefische/backend/IdServiceTest.java:11]: Mocking the idService in the IdServiceTest misses the point of testing I think. You could invoke the real method of the idService and check if the result is a string of a length greater than zero.

Comment (Satttoshi) [reply backend/src/test/java/de/neuefische/backend/IdServiceTest.java:11]: You mean you actually dont have to create a testing class for each class and can test it via where its being invoked?

**Pull request merged**